include/inexor/vulkan-renderer/wrapper/mesh_buffer.hpp

```diff
@@ -25,30 +26,28 @@ namespace inexor::vulkan_renderer::wrapper {
 class MeshBuffer {
 
 private:
-    std::string m_name = "";
-
+    const wrapper::Device &m_device;
+    std::string m_name;
```

Comment:
```suggestion
    const std::string m_name;
```

The names can be const in generell, just saw that many other use non const yet. But they will be never changed?

Reply:
These actually can't be `const`, because of the move constructors.

Reply:
Done.